scripts/variantstore/wdl/GvsExtractCohort.wdl

```diff
@@ -0,0 +1,135 @@
+version 1.0
+
+import "GvsPrepareCallset.wdl" as GvsPrepareCallset
+import "GvsExtractCallset.wdl" as GvsExtractCallset
+
+# Workflow used by AoU to extract variants for a given cohort of sample_names
+
+workflow GvsExtractCohort {
+
+  input {
+    File cohort_sample_names
+    String query_project
+    String gvs_dataset
+    String gvs_extraction_cohorts_dataset
+    String gvs_extraction_destination_dataset
+    String gvs_extraction_temp_tables_dataset
+    String extraction_uuid
+    String output_gcs_dir
+
+    # Extract parameters
+    File wgs_intervals
+    Int scatter_count
+
+    File reference
+    File reference_index
+    File reference_dict
+
+    String output_file_base_name
+
+    String? fq_filter_set_table
+    String? filter_set_name
+    File? gatk_override
+  }
+
+  call CreateCohortSampleTable {
+    input:
+      cohort_sample_names = cohort_sample_names,
+      gvs_dataset = gvs_dataset,
+      gvs_extraction_cohorts_dataset = gvs_extraction_cohorts_dataset,
+      query_project = query_project,
+      table_name = extraction_uuid
+  }
+
+  call GvsPrepareCallset.GvsPrepareCallset {
+    input:
+      destination_cohort_table_name   = extraction_uuid,
+      data_project                    = query_project,
+      default_dataset                 = "", # unused if fq_* args are given
+      fq_petvet_dataset               = gvs_dataset,
+      fq_cohort_sample_table          = CreateCohortSampleTable.fq_cohort_sample_table,
+      fq_sample_mapping_table         = "~{gvs_dataset}.sample_info",
+      fq_temp_table_dataset           = gvs_extraction_temp_tables_dataset,
+      fq_destination_dataset          = gvs_extraction_destination_dataset
+  }
+
+
+  call GvsExtractCallset.GvsExtractCallset {
+    input:
+      data_project = "", # unused if fq_filter_set_* args are given or filtering is off
+      query_project = query_project,
+      default_dataset = "", # unused if fq_filter_set_* args are given or filtering is off
+      filter_set_name = "", # unused if fq_filter_set_* args are given or filtering is off
```

Comment:
Does AoU want unfiltered data, that would be very surprising

Reply:
I don't think so. I hadn't gotten around to adding the filter steps yet which is the main reason these are missing. Do you know what the process for that will be? Do we just point the filter set arguments to tables that you guys generate or is it something that is created on a per extract basis?